composer.json

```diff
@@ -128,7 +128,6 @@
         "post-install-cmd": "@phing-install-dependencies",
         "post-update-cmd": "@phing-install-dependencies",
         "qa": "phing qa-console -Ddefaultconfigs=true",
-        "install-npm-dependencies": "npm install && phing copynodemodules",
-        "update-npm-dependencies": "npm update && phing copynodemodules"
+        "install-npm-dependencies": "npm install && phing copynodemodules"
```

Comment:
I did not make this change (intentionally), it either came from upstream or from installing node locally...but it matches what's currently in `dev` and has been for years...

Reply:
@EreMaijala added that new line in #3073. Is it possible you accidentally undid his change when resolving merge conflicts?

Reply:
Argh thank you @demiankatz .  I didn't see it in the git history for that file because I was accidentally looking at a commit-specific history.  I will revert my accidental change.

Reply:
Fixed